Projections/Benchmark/Benchmark.csproj

```diff
@@ -25,48 +13,11 @@
     <ProjectReference Include="..\Windows\Windows.csproj" />
   </ItemGroup>
 
-  <Target Name="GenerateProjection" Condition="'$(GenerateTestProjection)' == 'true'">
-    <ItemGroup>
-      <!--Do not publish projection source winmds -->
-      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" Condition="%(ReferenceCopyLocalPaths.Filename) == 'BenchmarkComponent'" />
-    </ItemGroup>
-    <PropertyGroup>
-      <CsWinRTVerbosity>high</CsWinRTVerbosity>
-      <CsWinRTResponseFile>$(GeneratedFilesDir)cswinrt_benchmark.rsp</CsWinRTResponseFile>
-      <CsWinRTCommand>$(CsWinRTExe) %40"$(CsWinRTResponseFile)"</CsWinRTCommand>
-    </PropertyGroup>
-    <PropertyGroup>
-      <CsWinRTParams>
--verbose
--target $(TargetFramework)
--in 10.0.18362.0
--in @(ReferenceWinMDs->'"%(FullPath)"', ' ')
--out "$(GeneratedFilesDir.TrimEnd('\'))"
+  <PropertyGroup>
+    <CsWinRTFilters>
 -exclude Windows
```

Comment:
Should we make excluding Windows namespace the default and have a flag to override that behavior?  I feel like almost everyone other than the Windows SDK projection would exclude windows and will currently need to know to exclude it.

Reply:
that is now the behavior (you probably weren't reviewing the latest commit)

Reply:
my mistake - current commit should show new behavior